File: Persona.py
from abc import ABC, abstractmethod


class Persona(ABC):
  def __init__(self, tipoId= '', id= '', nombre= '', tipoPersona= '', municipio= '', departamento= '', direccion= '', correo= '', tel= '') -> None:
    self._tipoId = tipoId
    self._id = id
    self._nombre = nombre
    self._tipoPersona = tipoPersona
    self._municipio = municipio
    self._departamento = departamento
    self._direccion = direccion
    self._correo = correo
    self._tel = tel
  
  @abstractmethod
  def getId(self):
    return self._id

File: Kontab/src/core/Persona/Persona.js
export default class Persona {
    id;
    nombre;
    pais;
    ciudad;
    direccion;
    correo;
    telefono;

    constructor(id, nombre, pais, ciudad, direccion, correo, telefono){
      this.id = id;
      this.nombre = nombre;
      this.pais = pais;
      this.ciudad = ciudad;
      this.direccion = direccion;
      this.correo = correo;
      this.telefono = telefono;
    }
}